src/core/core_structs/operations_problem.jl

```diff
@@ -78,44 +78,44 @@ into the optimization model the default is nothing.
 if false it runs for one time step
 -`initial_time::Dates.DateTime = PSY.get_forecasts_initial_time(sys)`: initial time of forecast
 """
-function OperationModel(::Type{M},
-                        model_ref::ModelReference,
+function OperationsProblem(::Type{M},
+                        model_ref::FormulationTemplate,
```

Comment:
Just a comment:  it now appears odd to have a variable called model_ref that is of type FormulationTemplate, which you changed because the team didn't like the term ModelReference.

The same comment applies to lots of other variables.